### ISSUE TITLE
Westend halted 5,478,664 [Thread 'tokio-runtime-worker' panicked at 'Storage root must match that calculated.', ]

### ISSUE THREAD

**Issue by jun0tpyrc** [paritytech/polkadot#2973]
```
Thread 'tokio-runtime-worker' panicked at 'Storage root must match that calculated.', /usr/local/cargo/git/checkouts/substrate-7e08433d4c370a21/3655f9b/frame/executive/src/lib.rs:455

This is a bug. Please report it at:

	https://github.com/paritytech/polkadot/issues/new

2021-05-04 04:26:42  panicked at 'Storage root must match that calculated.', /cargo-home/git/checkouts/substrate-7e08433d4c370a21/3655f9b/frame/executive/src/lib.rs:455:9
2021-05-04 04:26:42  Block prepare storage changes error:
RuntimeApiError(Application(Execution(Other("Wasm execution trapped: wasm trap: unreachable\nwasm backtrace:\n    0: 0x2124 - <unknown>!rust_begin_unwind\n    1: 0x1ee1 - <unknown>!core::panicking::panic_fmt::hfa15f5472ef5e557\n    2: 0x1427 - <unknown>!core::panicking::panic::hc64f9753b8a1be76\n    3: 0xe06af - <unknown>!Core_execute_block\nnote: run with `WASMTIME_BACKTRACE_DETAILS=1` environment variable to display more information\n"))))
2021-05-04 04:26:42  💔 Error importing block 0x66535350bf0d031dc15a0b6ee20165c0cd7ea49fec85685915f732e24d69e141: Err(Other(ClientImport("Import failed: Error at calling runtime api: Execution failed: Other(\"Wasm execution trapped: wasm trap: unreachable\\nwasm backtrace:\\n    0: 0x2124 - <unknown>!rust_begin_unwind\\n    1: 0x1ee1 - <unknown>!core::panicking::panic_fmt::hfa15f5472ef5e557\\n    2: 0x1427 - <unknown>!core::panicking::panic::hc64f9753b8a1be76\\n    3: 0xe06af - <unknown>!Core_execute_block\\nnote: run with `WASMTIME_BACKTRACE_DETAILS=1` environment variable to display more information\\n\")")))
```

tested on own 0.8.28-1 & 0.8.30 issue persisting , cannot finalize block

same as result shown in  explorer like https://westend.subscan.io/

**Comment by bkchr** [comment]
Yeah, something went wrong. We are working on a fix. This will very likely include that you need to resync.

**Comment by gituser** [comment]
@bkchr any update on this? 

Can't transfer anything on Westend (transaction is stuck forever in the pool), but block height is increasing.

**Comment by andresilva** [comment]
This is fixed now, make sure you update to v0.9.0-rc3 to keep syncing Westend.

**Comment by gituser** [comment]
Seems to be working on `v0.9.0-rc3`.

Also if someone is using https://github.com/paritytech/substrate-api-sidecar don't forget to update to the latest tag `v5.0.1` otherwise it won't work for new blocks.
